frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -24,26 +30,45 @@ import { VMIKind, VMKind } from '../../types';
 import { getMigrationVMIName, isMigrating } from '../../selectors/vmi-migration';
 import { getBasicID, getLoadedData, getResource } from '../../utils';
 import { getVMStatus } from '../../statuses/vm/vm';
+import { getVmiIpAddresses } from '../../selectors/vmi';
 import { vmStatusFilter } from './table-filters';
 import { menuActions } from './menu-actions';
 
 import './vm.scss';
 
+const VMIPlaceHolder: React.FC<VMIPlaceHolderProps> = ({ kind, name }) => (
+  <span>
+    <ResourceIcon className="kubevirt-vmi-placeholder-label" kind={kind} />
+    <span className="co-resource-item__resource-name kubevirt-vmi-placeholder" data-test-id={name}>
+      {name}
+    </span>
+  </span>
+);
+
 const tableColumnClasses = [
-  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
-  classNames('col-lg-4', 'col-md-4', 'hidden-sm', 'hidden-xs'),
-  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  classNames('col-lg-2', 'col-md-2', 'col-sm-6', 'col-xs-6'),
```

Comment:
@matthewcarleton hi, I'm doing here a "14 spans" size tabls ... it works ... is it OK ?

Reply:
Ref: https://www.patternfly.org/v4/documentation/react/layouts/grid